Gemfile

```diff
@@ -23,4 +23,8 @@ group(:dev) do
   gem 'guard'
   gem 'guard-rspec'
   gem 'ruby_gntp'
+  gem 'chefspec'
+  gem 'rubocop'
+  gem 'knife-spork'
+  gem 'kitchen-vagrant'
```

Comment:
We should check git blame before changing anything. I vaguely remember changing a project to avoid the magic behavior but I can't recall if it's this or not. 


> On Sep 11, 2015, at 8:32 PM, Lamont Granquist <notifications@github.com> wrote:
> 
> In Gemfile:
> 
> > @@ -23,4 +23,8 @@ group(:dev) do
> >    gem 'guard'
> >    gem 'guard-rspec'
> >    gem 'ruby_gntp'
> > +  gem 'chefspec'
> > +  gem 'rubocop'
> > +  gem 'knife-spork'
> > +  gem 'kitchen-vagrant'
> that would be the question... it looks like its a mistake to me and its not intended to install those normally.... but... shrug... needs a comment if its deliberate...
> 
> —
> Reply to this email directly or view it on GitHub.
>